content/en/basics/how-retrieval-works/basic-retrieval/index.md

```diff
@@ -17,11 +17,7 @@ aliases:
 
 ## Lassie
 
-Lassie is a simple retrieval client for IPFS and Filecoin. It finds and fetches your data over the best retrieval protocols available. Lassie makes Filecoin retrieval easy. While Lassie is powerful, the core functionality is expressed in a single CLI command:
-
-```shell
-lassie fetch <CID>
-```
+Lassie is a simple retrieval client for IPFS and Filecoin. It finds and fetches your data over the best retrieval protocols available. Lassie makes Filecoin retrieval easy. While Lassie is powerful, the core functionality is expressed in a single CLI command - `lassie fetch <CID>`
```

Comment:
Revert this back to the code-block version so users can click the big copy button if they want.

Reply:
This is based on the feedback from users. Having that command well before the installation steps keep tripping them up. Most of the newcomers have complained that it is bad form and it should be after the installation instructions. Another thing to note here is that that was not really a command but just a demonstration of how powerful this tool is. 
@smagdali can probably give you more details on the user experience. This PR is based on the feedback his team collected.

Reply:
Ok, thanks for clarifying. I've removed the command altogether.